js/leaderboard.js

```diff
@@ -70,22 +70,33 @@
       const existingNicknameEntry = result.find(function(existingEntry) {
         return existingEntry.name === entry.Nickname;
       });
-
+      
       if (existingNicknameEntry) {
+        // only count the same domain once
+        const exsitingDomain = existingNicknameEntry.entries.find((existingEntry) => {
```

Comment:
typo ;)

Reply:
oh yes indeed, feel free to keep them if you find any mode ;)